Plugin/MirasvitCreditQuotePaymentImportDataBeforePlugin.php

```diff
@@ -52,7 +52,7 @@ public function __construct(DiscountHelper $discountHelper)
      */
     public function beforeExecute(ObserverInterface $subject, Observer $observer)
     {
-        if ($this->discountHelper->isMirasvitAdminQuoteUsingCreditObserver($observer)) {
+        if ($this->eventsForThirdPartyModules->runFilter("checkMirasvitCreditAdminQuoteUsed", '', $observer)) {
```

Comment:
Nice idea! But can we use parameter `false` instead of `''`

Reply:
fixed